pandas/core/frame.py

```diff
@@ -8936,6 +8936,10 @@ def append(
 
         Columns in `other` that are not in the caller are added as new columns.
 
+        .. deprecated:: 1.4.0
+            append is deprecated,
```

Comment:
I don't know if we need 'append is deprecated'. This IS the append doc-string. We don't do this anywhere else. 

Also capital the first letter of the sentence.

Reply:
@jreback isn't it the same as in `lookup` https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.lookup.html , where it says

> DataFrame.lookup is deprecated

?

Reply:
I took inspiration from existing messages that describe the issue. Since append refers to the method, I would not capitalize it.

Reply:
sure lookup is a full message, this is not.

Reply:
i am ok if its a full sentence in a similar way.

Reply:
What about https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.slice_shift.html?

Reply:
@gesoos I think if you make it similar to how `lookup` is (suggest to use concat instead, and point to the relevant part of the user guide if there is one - check the whatsnew note) it'll be OK

Reply:
yeah we would like to make these consistent.